src/main/java/com/toopher/AuthenticationRequest.java

```diff
@@ -0,0 +1,132 @@
+package com.toopher;
+
+import org.apache.http.NameValuePair;
+import org.apache.http.message.BasicNameValuePair;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Provides information about the status of an authentication request
+ * 
+ */
+public class AuthenticationRequest extends ApiResponseObject {
+    /**
+     * The ToopherAPI object associated with this authentication request
+     */
+    public ToopherApi api;
+
+	/**
+     * The unique id for the authentication request
+     */
+    public String id;
+
+    /**
+     * Indicates if the user has responded to the authentication request
+     */
+    public boolean pending;
+
+    /**
+     * Indicates if the authentication request was granted
+     */
+    public boolean granted;
+
+    /**
+     * Indicates if the authentication request was automated
+     */
+    public boolean automated;
+
+    /**
+     * Indicates the reason (if any) for the authentication request's outcome
+     */
+    public String reason;
+
+    /**
+     * Indicates the code associated with the reason for the authentication request's outcome
+     */
+    public int reasonCode;
+
+    /**
+     * The UserTerminal associated with the authentication request
+     */
+    public UserTerminal terminal;
```

Comment:
Could you look into using links to the objects? It's something like `{@link com.toopher.UserTerminal}`